components/AllocatedCases/AllocatedCases.tsx

```diff
@@ -17,7 +17,7 @@ const AllocatedCases = ({ id }: Props): React.ReactElement => {
   }
   return (
     <>
-      {allocations.workers.length === 0 ? (
+      {allocations.workers?.length === 0 ? (
```

Comment:
I think we can simplify this conditional render (even if the problem is not related to this PR)